src/languages/python.js

```diff
@@ -120,22 +121,32 @@ export default function(hljs) {
     'False',
     'None',
     'NotImplemented',
-    'True',
+    'True'
+  ];
+
+  const TYPES = [
+    "Dict",
+    "List",
+    "Coroutine",
+    "Any"
   ];
```

Comment:
I know we don't usually go for an exhaustive list for things like this, but would you want to add a few more to this list? My suggestion would be to add: `Generic`, `Optional`, `Sequence`, `Set`, `Tuple`, `Type`, `Union`.

However, could at least match `-> SomeType:` instead and highlight what's in between `->` and `:`?

![image](https://user-images.githubusercontent.com/1246453/110004502-5e2b4a00-7ccc-11eb-8ac9-84266492bac9.png)

Reply:
Matching _anything_ between would be inconsistent since right now `int` in `List[int]` is a built-in, not a type... we'd have to try to match just the camel case?  

Im fine with extending the list for now though.  Do you know python or did you get those from a list somewhere?

Reply:
> Matching _anything_ between would be inconsistent since right now `int` in `List[int]` is a built-in, not a type... we'd have to try to match just the camel case?

I'd prefer with matching camel case. It'd allow highlighting custom types that code snippets could have; like an arbitrary `UserClass` e.g.

> Im fine with extending the list for now though. Do you know python or did you get those from a list somewhere?

Yea, I just pulled those types from a project I use at work.

Reply:
But they are python built ins not custom from your project, right? :)

Reply:
Technically, only the lower case `str`, `int`, `bool`, etc. are "built-ins" according to Python. The ones I listed and the ones you have in your PR currently are imported from the `typing` package that's a part of the Python STD. But yes, the ones I listed would be classified as built-ins for our purposes

Reply:
I buffed up the list.